File: backend-nest/scripts/clean-tables.js
const { Client } = require('pg');

const client = new Client({
  host: 'localhost',
  port: 5432,
  user: 'ssto',
  password: 'sstopass',
  database: 'sstodb'
});

async function cleanTables() {
  try {
    await client.connect();
    console.log('Подключено к БД как ssto');
    
    const res1 = await client.query('DROP TABLE IF EXISTS signals CASCADE');
    console.log('Таблица signals удалена');
    
    const res2 = await client.query('DROP TABLE IF EXISTS requests CASCADE');
    console.log('Таблица requests удалена');
    
    console.log('Все таблицы очищены!');
  } catch (err) {
    console.error('Ошибка:', err.message);
  } finally {
    await client.end();
  }
}

cleanTables();

File: merge-backup-20250927-113855/Projects/test-ssto-project/frontend-static/backend-nest/scripts/clean_tables.py
# Создаем Python скрипт
@'
import psycopg2

try:
    # Подключение к БД
    conn = psycopg2.connect(
        host="localhost",
        port=5432,
        database="sstodb",
        user="ssto",
        password="sstopass"
    )
    
    cur = conn.cursor()
    print("Подключено к БД")
    
    # Удаляем таблицы
    cur.execute("DROP TABLE IF EXISTS signals CASCADE")
    print("Таблица signals удалена")
    
    cur.execute("DROP TABLE IF EXISTS requests CASCADE")
    print("Таблица requests удалена")
    
    # Применяем изменения
    conn.commit()
    print("Все таблицы очищены!")
    
except Exception as e:
    print(f"Ошибка: {e}")
    
finally:
    if cur:
        cur.close()
    if conn:
        conn.close()
'@ | Out-File -FilePath scripts\clean_tables.py -Encoding UTF8